applications/FluidDynamicsApplication/python_scripts/compute_drag_process.py

```diff
@@ -72,10 +73,15 @@ def ExecuteInitialize(self):
 
                 output_file_name = self.params["model_part_name"].GetString() + "_drag.dat"
 
-                file_handler_params = KratosMultiphysics.Parameters('''{ "output_file_name" : "" }''')
-
+                file_handler_params = self.params.Clone()
+                file_handler_params.AddEmptyValue("output_file_name")
                 file_handler_params["output_file_name"].SetString(output_file_name)
-                file_handler_params.AddValue("write_buffer_size", self.params["write_buffer_size"])
+
+                file_handler_params.RemoveValue("model_part_name")
```

Comment:
I think the safer option would be to start from an empty object and copy only the params needed by the AsciiFileWriter
I guess this will be fewer lines of code plus if new parameters are added in the dragprocess one does not have to delete them manually

Reply:
Ok, updated. It is indeed less.